### DISCUSSION TITLE
Is there a way to get a warning if the coordinator goes offline?

### DISCUSSION THREAD

**Discussion by rocket59** [Koenkk/zigbee2mqtt#14391] [Devices]
My z2m network now runs as a homeassistant addon, with HA being installed in a Proxmox VM. For some unknown reason the USB passthrough to the VM stopped working the other day (and needed a reboot) and of course nothing worked. Is there a way to be warned if the coordinator itself is offline/not connected rather than rely on device availability?

**Comment by hmoffatt** [comment]
The Home Assistant entity `sensor.zigbee2mqtt_bridge_state` should tell you if zigbee2mqtt is connected to MQTT or not.

**Reply by rocket59** [reply]
Thanks! Will check that out.
